plugin-server/src/ingestion-schema.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ *
+ * written to kafka topic: clickhouse_session_replay_events
+ * and from there to clickhouse table: session_replay_events
+ */
+export interface SummarizedSessionRecordingEvent {
```

Comment:
Should we just have this in the general types file? I think we moved all the other types there.

Reply:
It was so I could generate types only from this file to avoid having a massive schema from day one